### PR TITLE
set voltage script

### PR DESCRIPTION
**Pull request by inescalier**

### CODE REVIEW

File: arduino/main/src/setVoltage/setVoltage.ino [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#include <Wire.h>
+
+#define MAX5434_VREG_COMMAND 17  //write ti volatile register ; wiper position update
+#define MAX5434_NVREG_COMMAND 33  //write to non volatile register ; wiper position unchanged
```
Comment (lbo462) [inline arduino/main/src/setVoltage/setVoltage.ino:4]: Is this used ?

File: arduino/main/src/setVoltage/setVoltage.ino [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#define MAX5434_NVREGXVREG_COMMAND 97  //byte transfert from non volatile register to volatile register ; wiper position update
```
Comment (lbo462) [inline arduino/main/src/setVoltage/setVoltage.ino:5]: Same here

**Review by MarsiaSec — CHANGES_REQUESTED** [review]
add a read me and fix the comments plz. Thank you 😘

File: arduino/main/src/setVoltage/setVoltage.ino [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    delay(2000);
+}
+
+void loop(){
+    analogWrite(motorLM, 70) ;   
+    analogWrite(motorLP, 0);   
+    analogWrite(motorRP, 70);   
+    analogWrite(motorRM, 0);   
+
+    Serial.println(3.0 * analogRead(A7) * (5.0/1023.0));
```
Comment (lbo462) [inline arduino/main/src/setVoltage/setVoltage.ino:31]: What's on A7 ? Maybe adding a comment about this line might be useful

**Review by lbo462 — CHANGES_REQUESTED** [review]
Adding a README would be nice
Plus, you can remove the pasted comments, there are not that explicit,
And you could even write your own

File: arduino/main/src/setVoltage/setVoltage.ino (hunk 1 above)
Comment (inescalier) [reply arduino/main/src/setVoltage/setVoltage.ino:4]: 2 are used, 2 are not for now. Still good to have them if needed

**Review by lbo462 — APPROVED** [review]
Kool

**Review by MarsiaSec — CHANGES_REQUESTED** [review]

**Review by MarsiaSec — APPROVED** [review]

**Pull request merged**